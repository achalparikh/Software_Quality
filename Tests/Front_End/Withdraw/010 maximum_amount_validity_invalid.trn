login
standard 
ACHAL PARIKH
withdraw 
499.95	
logout