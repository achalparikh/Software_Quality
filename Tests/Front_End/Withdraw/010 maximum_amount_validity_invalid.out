Enter user type - "standard" or "admin": 
Logging in as standard user - enter name:
You are now logged in as "ACHAL PARIKH"
COMMAND: withdraw - enter amount:
ERROR: amount cannot be withdrawn. 
You are now logged out
